### DISCUSSION TITLE
Some drivers???

### DISCUSSION THREAD

**Discussion by CPU-GPU** [CodeRunner5235/Opencore-Steam-Deck#7] [Q&A]
I recently managed to install MacOS Monterey on my steam deck and I was wondering if there are any drivers for it... @CodeRunner5235

**Comment by CodeRunner5235** [comment]
There are no drivers for the GPU on the steamdeck at this point in time

**Reply by CPU-GPU** [reply]
@CodeRunner5235 not even sound or internet drivers? I tried to find them on the internet but with no success...

**Reply by CodeRunner5235** [reply]
No sound drivers yet. Wifi and bluetooth dont work either. Type C ethernet adapters and hubs should work fine if it works with macOS
